WeatherList.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useLayoutEffect } from "react";
+import { Button, StyleSheet, Text, View } from "react-native";
+import { useNavigation } from "@react-navigation/native";
+import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
+import FiveDaysScreen from "./FiveDaysScreen";
+import SevenDaysScreen from "./SevenDaysScreen";
+import Ionicons from "@expo/vector-icons/Ionicons";
+import { StackNavigationProp } from "@react-navigation/stack";
+export type StackParamList = {
+  Home: undefined;
+  Details: { itemID: number; otherParam?: string };
+  Modal: undefined;
+  Cool: undefined;
+};
+
+const Tab = createBottomTabNavigator();
+
+type HomeScreenNavigationProp = StackNavigationProp<StackParamList, "Home">;
+
+export function WeatherList() {
+  const navigation = useNavigation<HomeScreenNavigationProp>();
+
+  useLayoutEffect(() => {
+    navigation.setOptions({
+      tabBarBadge: 1,
+    });
+  });
+
+  return (
+    <Tab.Navigator
+      screenOptions={({ route }) => ({
+        tabBarIcon: ({ focused, color, size }) => {
+          let iconText: string;
+
+          if (route.name === "Five Days") {
+            iconText = "5";
+          } else if (route.name === "Seven Days") {
+            iconText = "7";
+          }
+
+          return <Text style={{ color, fontSize: size }}>{iconText}</Text>;
+        },
+        tabBarActiveTintColor: "tomato",
+        tabBarInactiveTintColor: "gray",
+      })}
+    >
+      <Tab.Screen name="Five Days" component={FiveDaysScreen} />
```

Comment:
The Five Days and Seven Days screens should each use the same component and use the `initialParam` prop to set the number of days. Copy and pasting the code twice is not a good practice. 

Here is the requirement from the assignment
"Your forecast component should be reused for each tab. What controls the # of days should be an `initialParam` which you can find in the React Navigation docs."